installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -105,4 +106,34 @@ You can use dedicated or shared IFLs to assign sufficient compute resources. Res
 
 include::modules/install-sno-ibm-z.adoc[leveloffset=+2]
 
-include::modules/install-sno-ibm-z-kvm.adoc[leveloffset=+2]
\ No newline at end of file
+include::modules/install-sno-ibm-z-kvm.adoc[leveloffset=+2]
+
+[id="installing-sno-with-ibmpower"]
+== Installing {sno} with {ibmpowerProductName}
+
+Installing a single-node cluster on {ibmpowerProductName} requires user-provisioned installation using either the "Installing a cluster with {op-system-base} KVM on {ibmpowerProductName}" procedure.
```

Comment:
```suggestion
Installing a single-node cluster on {ibmpowerProductName} requires user-provisioned installation using the "Installing a cluster with {ibmpowerProductName}" procedure.
```

Reply:
Updated as per Silke's suggestion above.

Reply:
@stevsmit I found only one instance which Silke has pointed above. Could you please check and confirm ? Thanks.